libphal/phal_sbe.C

```diff
@@ -20,6 +20,29 @@ using namespace openpower::phal;
 using namespace openpower::phal::utils::pdbg;
 using namespace openpower::phal::pdbg;
 
+/**
+ * @brief helper function to log SBE debug data
+ *
+ * @param[in] proc processor target to operate on
+ */
+void logSbeDebugData(struct pdbg_target *proc)
+{
+	std::array<const uint32_t, 5> cfamAddr = {0x2986, 0x2809, 0x282A,
+						  0x2829, 0x1007};
+	uint32_t val = 0xFFFFFFFF; // Invalid value
```

Comment:
this should go inside the for loop, 
first register if we get correct value then it will e set to that value and if second register fails we will not know

Reply:
> this should go inside the for loop, first register if we get correct value then it will e set to that value and if second register fails we will not know

done